nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -418,6 +418,14 @@ in {
         '';
       };
 
+      extraSecrets = lib.mkOption {
+        type = lib.types.attrs;
+        default = {};
+        description = ''
+          Files containing extra secret environment variables to pass to all mastodon services.
```

Comment:
From your code I'm inferring that the name of the file must be the name of the environment variable? Can you expand on that here in the description?

Reply:
E.g. by adding an example value